toml/src/main/kotlin/org/rust/toml/completion/LocalCargoTomlDependencyCompletionProvider.kt

```diff
@@ -54,16 +53,13 @@ class LocalCargoTomlDependencyCompletionProvider : TomlKeyValueCompletionProvide
 
         val indexService = CratesLocalIndexService.getInstance()
 
-        val versions = indexService.getCrate(name)?.versions ?: return
+        val versions = indexService.getCrate(name)?.sortedVersions ?: return
         val elements = versions.mapIndexed { index, variant ->
-            val lookupElement = LookupElementBuilder.create(variant.version)
+            val lookupElement = LookupElementBuilder.create(variant.versionString)
                 .withInsertHandler(StringValueInsertionHandler(keyValue))
                 .withTailText(if (variant.isYanked) " yanked" else null)
 
-            PrioritizedLookupElement.withPriority(
-                lookupElement,
-                index.toDouble()
-            )
+            PrioritizedLookupElement.withExplicitProximity(lookupElement, index)
         }
         result.addAllElements(elements)
```

Comment:
Discussed in person the following problem with sorting:
![image](https://user-images.githubusercontent.com/4854600/105862203-95894580-6000-11eb-9824-bb04abffdca4.png)
Here, `0.2.1` should go after `0.2.10`, `0.2.9`, and so on. The problem is due to the `liftShorter` weigher: looks like it has a higher priority than others (see `com.intellij.codeInsight.completion.BaseCompletionService#defaultSorter` for more detail). To avoid this situation, let's try using an empty sorter:
```
result.withRelevanceSorter(CompletionSorter.emptySorter()).addAllElements(elements)
```